example/simple_filter.cpp

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright 2005-2007 Adobe Systems Incorporated
```

Comment:
What is this files about?
Why you added file that is not copyrighted by yourself?